sections/telemetry_system.tex

```diff
@@ -137,6 +137,21 @@ \subsubsection{Magnetometer}
 This magnetometer was selected because it has an \gls{i2c} interface and can be supplied at 3.3V.
 \cite{lsm6dso32-datasheet}
 
+\section{Board Design}
+
+In previous years the \gls{srad} board design was split over a number of boards to decrease coupling in case the
+designs needed to be reworked. These boards included a \gls{mcu} board, a sensor board, a power board, a radio board,
+and a \gls{cots} systems board, and led to the use of a "stack" of boards. This year, because previous designs were so
+simple for the sensor and power boards, \gls{cuinspace} decided to combine these boards with the \gls{mcu} board
+into a single combined design to save space in the rocket and money on board manufacturing costs. The radio board was
+kept seperate because of the frequency with which it is redesigned and so it could be replaced with a \gls{cots} radio
```

Comment:
I might change this to not use frequency (this made me think of RF when I read it and took me a couple reads to understand what you meant)